executor/modes/local/runner.py

```
@@ -286,6 +393,18 @@ async def _execute_task(self, task_data: Dict[str, Any]) -> None:
         # Create and initialize agent
         self.current_agent = ClaudeCodeAgent(task_data)
 
+        # Set callback to send heartbeat when Claude client is created
+        async def on_client_created():
+            try:
+                await self.websocket_client.send_heartbeat()
+                logger.info(
+                    f"[TaskStart] Sent heartbeat after Claude client created for task {task_id}"
+                )
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)